### PR TITLE
Gene Search "genus" Parameter

### PR DESCRIPTION
**Pull request by That-Thing**
Re-opening PR for the "genus" parameter since it never finished getting merged into main.   

The `genus` parameter allows a single genus to be automatically selected in the gene search form, which is useful for services like SoyBase which are centered only around the one genus.  
Example usage: `<lis-gene-search-element id="gene-search" genus="glycine"></lis-gene-search-element>`

### CODE REVIEW

File: dev/lis-gene-search-element.html
Comment (alancleary) [inline dev/lis-gene-search-element.html:35]: Rephrase "A search for a specific genus..." to "A search can be restrict to a specific genus..."

File: dev/lis-gene-search-element.html
Comment (alancleary) [inline dev/lis-gene-search-element.html:41]: Don't use this `genus` attribute by default since it's a specific use-case. Instead, mention in the text above that the search element is available in the console via the `geneSearchElement` variable and that the property can be applied to it that way.

File: src/lis-gene-search-element.ts
Comment (alancleary) [inline src/lis-gene-search-element.ts:175]: Remove the `id` attribute from this example since it's not being used.

File: src/lis-gene-search-element.ts
Comment (alancleary) [inline src/lis-gene-search-element.ts:188]: This is an optional property so define it as optional without a default value:
```typescript
genus?: string;
```

**Review by alancleary — CHANGES_REQUESTED** [review]
This PR needs some work. I left comments for minor things that you can take care of but there's some major issues that I think would be best solved via pair programming or if I fixed them myself. It's up to you.

Also, you need to regenerate the docs and commit the changes since you updated the doc strings

**Comment by That-Thing** [comment]
I have made the requested changes, and as for the major changes, I have no preference as to who makes them. I will leave that up to your discretion.

**Comment by alancleary** [comment]
> I have made the requested changes, and as for the major changes, I have no preference as to who makes them. I will leave that up to your discretion.

OK. For the sake of time I'll just push the changes.

**Comment by sammyjava** [comment]
Had to use `Glycine` not `glycine` as parameter value for it to work.
```
<lis-gene-search-element id="gene-search" genus="Glycine"></lis-gene-search-element>
```

**Review by sammyjava — APPROVED** [review]
Works for me on dev/examples/lis-gene-search-with-linkouts.html once I used `Glycine` rather than `glycine`.

**Review by alancleary — APPROVED** [review]
I'm approving the changes @That-Thing made based on my previous review.

**Comment by alancleary** [comment]
Let's hold off on merging this until there's a spec in place, which will hopefully be resolved at Wednesday's meeting.

**Pull request merged**
